btree/btree.c

```diff
@@ -6808,6 +6827,41 @@ static void btree_ut_kv_oper_thread_handler(struct btree_ut_thread_info *ti)
 			keys_put_count++;
 			key_first += ti->ti_key_incr;
 		}
+
+#if 0
+		/** Modify at least 20% of the values which have been inserted. */
+		key_first = key_iter_start;
+		while (key_first <= key_last) {
+			/**
+			 *  Embed the thread-id in LSB so that different threads
+			 *  will target the same node thus causing race
+			 *  conditions useful to mimic and test btree operations
+			 *  in a loaded system.
+			 */
+			key[0] = (key_first << (sizeof(ti->ti_thread_id) * 8)) +
+				 ti->ti_thread_id;
+			key[0] = m0_byteorder_cpu_to_be64(key[0]);
+			for (i = 1; i < ARRAY_SIZE(key); i++)
+				key[i] = key[0];
+
+			value[0] = ~key[0];
+			for (i = 1; i < ARRAY_SIZE(value); i++)
+				value[i] = value[0];
+
+			cred = M0_BE_TX_CB_CREDIT(0, 0, 0);
+			btree_callback_credit(&cred);
+
+			M0_BTREE_OP_SYNC_WITH_RC(&kv_op,
+						 m0_btree_put(tree, &rec,
+							      &ut_cb, 0,
+							      &kv_op, tx));
+			M0_ASSERT(data.flags == M0_BSC_SUCCESS);
+
+			keys_put_count++;
```

Comment:
No need to increment keys_put_count;

Reply:
Good catch. Copy+Paste error. Fixed it now.